packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -340,6 +385,42 @@ describe(Runner, () => {
         loggerMock.verifyAll();
     });
 
+    it('enqueue notification, send notification feature flag is enabled', async () => {
+        setupWebDriverCalls();
+
+        setupUpdateScanRunResultCall(getRunningJobStateScanResult());
+
+        scannerTaskMock
+            .setup(async s => s.scan(scanMetadata.url))
+            .returns(async () => Promise.resolve(passedAxeScanResults))
+            .verifiable();
+
+        setupGenerateReportsCall(passedAxeScanResults);
+        setupSaveAllReportsCall();
+        setupUpdateScanRunResultCall(getScanResultWithNoViolations());
+
+        serviceConfigurationMock.reset();
```

Comment:
either use strict mock for serviceConfigurationMock & notificationDispatcherMock

or add test for case where feature flag is not present / turned off.

Reply:
all other test cases are for features flag is turned off/ we set it up in beforeEach